### PR TITLE
Add Add-PnPFileSensitivityLabel cmdlet for assigning sensitivity labels to SharePoint files

### LINKED ISSUE #3135: Add-PnPFileSensitivityLabel
If possible it would be very helpful to assign Sensitivity Labels on files with PowerShell.

**Describe the solution you'd like**
Ideally it would be a simple cmdlet like Add-PnPFileSensitivityLabel that takes a path and then assigns a sensitivity label.

**Describe alternatives you've considered**
There are OOTB features for setting defaults on libraries and on containers, but there isn't really much in terms of files and I think most alternatives are covered with existing OOTB features or cmdets.

### PR DESCRIPTION
**Pull request by gautamdsheth**
## Type ##
- [] Bug Fix
- [x] New Feature
- [ ] Sample

## Related Issues? ##
Fixes #3135

## What is in this Pull Request ? ##
Added cmdlet to set the file sensitivity label information

### CODE REVIEW

File: src/Commands/Files/AddFileSensitivityLabel.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+namespace PnP.PowerShell.Commands.Files
+{
+    [Cmdlet(VerbsCommon.Add, "PnPFileSensitivityLabel")]
+    [RequiredApiDelegatedOrApplicationPermissions("graph/Files.ReadWrite.All")]
+    [RequiredApiDelegatedOrApplicationPermissions("graph/Sites.ReadWrite.All")]
+
+    public class AddFileSensitivityLabel : PnPGraphCmdlet
+    {
+        [Parameter(Mandatory = true, Position = 0, ValueFromPipeline = true)]
+        public string Url = string.Empty;
```
Comment (KoenZomers) [inline src/Commands/Files/AddFileSensitivityLabel.cs:17]: Could be nice to support an option with a FilePipeBind as well to be able to apply this to larger amounts of files?

File: documentation/Add-PnPFileSensitivityLabel.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### Example 1
+This example adds the sensitivity label information for the file at the specified URL.
+
+```powershell
+Add-PnPFileSensitivityLabel -Url "/sites/Marketing/Shared Documents/Report.pptx" -SensitivityLabelId b5b11b04-05b3-4fe4-baa9-b7f5f65b8b64 -JustificationText "Previous label no longer applies" -AssignmentMethod Privileged
+```
+
+This example adds the sensitivity label information for the file at the specified URL.
+
```
Comment (KoenZomers) [inline documentation/Add-PnPFileSensitivityLabel.md:39]: Can we have an example of how to remove the label as well, as that requires an empty string as I read?

File: src/Commands/Files/AddFileSensitivityLabel.cs (hunk 1 above)
Comment (gautamdsheth) [reply src/Commands/Files/AddFileSensitivityLabel.cs]: Fixed !

File: documentation/Add-PnPFileSensitivityLabel.md (hunk 2 above)
Comment (gautamdsheth) [reply documentation/Add-PnPFileSensitivityLabel.md:44]: Added another example to remove the label

**Pull request merged**